sros2/sros2/api/__init__.py

```diff
@@ -44,14 +44,16 @@
 HIDDEN_NODE_PREFIX = '_'
 DOMAIN_ID_ENV = 'ROS_DOMAIN_ID'
 
-NodeName = namedtuple('NodeName', ('node', 'ns', 'fqn'))
-TopicInfo = namedtuple('Topic', ('fqn', 'type'))
+NODE_NAME = namedtuple('NodeName', ('node', 'ns', 'fqn'))
```

Comment:
Why are you formatting this typename like a constant?

Reply:
My bad, there's no reason to do so :+1:, I'll just move the new constant above these types declaration to keep the constants grouped together